crates/store/src/buffer.rs

```diff
@@ -197,7 +199,12 @@ impl BufferStorage {
     pub fn set_page_erases(&mut self, page: usize, cycle: usize) {
         self.page_erases[page] = cycle;
     }
+}
 
+/// Error indicating that an operation was interrupted.
+pub const INTERRUPTION: Error = Error::new_const(Space::World as u8, 0xffff);
```

Comment:
We could actually make this `Space::Internal`. Not sure though what's best.

Reply:
Not really, because it would change space between when it's produced and when it's compared. What we could do though is expose the popped error and generate the internal one. I'll test it.

Reply:
It works. I agree it's better.